R/reporter-progress.R

```diff
@@ -47,6 +48,7 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
       self$show_praise <- show_praise
       self$min_time <- min_time
       self$update_interval <- update_interval
+      self$frames <- cli::get_spinner()$frames
```

Comment:
Need to store spinner frames on init, since that's what should determine the unicode status.

Reply:
This is enough for the progress reporter, but in general might be not. With the current setup the `add_result` handler runs in the reproducible context, which is correct. We might be to get away with it, so we don't necessarily need to do anything now, just don't forget this. :) Maybe add a comment somewhere?

Reply:
Hmmm, to fix this probably I think we'd need to split `test_code()` into two pieces — one piece would run the code and catch the expectations (with `local_test_context()`), and then the other would send the expectations to the reporter. I'll turn this into an issue and think about more later on.